client/my-sites/checkout/composite-checkout/contact-validation.js

```diff
@@ -33,6 +35,14 @@ const wpcomValidateDomainContactInformation = ( ...args ) =>
 			{ apiVersion: '1.2' }
 		);
 	} );
+async function wpcomValidateSignupEmail( ...args ) {
+	try {
+		const response = await wpcom.validateNewUser( ...args, null );
```

Comment:
It's not very important, but I _think_ that you could change this function to:

```js
async function wpcomValidateSignupEmail( ...args ) {
  return wpcom.validateNewUser( ...args, null );
}
```

...since if `wpcom.validateNewUser` rejects or throws, that will cause this function to reject, and because `throw` in an async function is the same as a rejection.

Reply:
Good point, I've made this change.